docs/release-notes/next-hc/RELEASE-NOTES-0.9.9.md

```diff
@@ -0,0 +1,90 @@
+# Hyperchains
+
+This is release of Hyperchains as described in [Periodically Syncing HyperChains](https://github.com/aeternity/hyperchains-whitepaper/blob/master/Periodically-Syncing-HyperChains.md).
+See also [aeternity Hyperchains](https://aeternity.com/hyperchains).
+
+Hyperchains integrate with other blockchain networks, leveraging their security and consensus mechanisms while providing scalable, energy-efficient hyperchains.
+
+An aeternity Hyperchain comes with all the normal features of aetenity:
+* FATE VM: Efficient virtual machine for executing smart contracts.
+* Sophia Language: A functional programming language for secure smart contracts.
+* State Channels: Off-chain scaling for private and secure transactions.
+* AENS: Æternity Naming System for human-readable addresses.
+* Oracles: Bridging real-world data into the blockchain ecosystem.
+
+See [aeternity.com](aeternity.com) for more information.
+
+# About this release
+
+This preliminary implementation of Hyperchains lays the foundation for a scalable and energy-efficient blockchain system. Key features introduced in this release include:
+
+## Core Features
+1. **Staking Mechanism**
+   - Enables participants to stake tokens and qualify for block production and validation. The staking framework ensures fairness and rewards based on contributions and performance.
```

Comment:
You don't need to qualify for validation (not even if you call the producers validators).